File: src/dataforms/app/form/SideMenuForm.java
package dataforms.app.form;

import dataforms.menu.Menu;
import dataforms.menu.SideMenu;

/**
 * サイドメニューフォームクラス。
 *
 */
public class SideMenuForm extends MenuForm {
	/**
	 * コンストラクタ。
	 */
	public SideMenuForm() {
		super(null);
	}

	@Override
	protected Menu newMenuComponent() {
		SideMenu menu = new SideMenu();
		return menu;
	}


}

File: HTML/META-INF/resources/frame/default/SideMenuForm.js
/**
 * メニューフォーム.
 */

createSubclass("SideMenuForm", {}, "MenuForm");
/**
 * ページの各エレメントとの対応付け.
 */
SideMenuForm.prototype.attach = function() {
	Form.prototype.attach.call(this);
	this.menu = this.newInstance(this.sideMenu);
	this.menu.init();
	this.menu.attach();
};
